File: GoogleCheckout.Source/examples/notification_sender/TrustAllCertificatePolicy.cs
using System;
using System.Net;
using System.Security.Cryptography.X509Certificates;

namespace NotificationSender
{
  public class TrustAllCertificatePolicy : System.Net.ICertificatePolicy {
    public TrustAllCertificatePolicy()
    {}

    public bool CheckValidationResult(ServicePoint sp,
      X509Certificate cert,WebRequest req, int problem) {
      return true;
    }
  }
}

File: GoogleCheckout.Source/examples/vb.net/notification_sender/TrustAllCertificatePolicy.vb
Imports System
Imports System.Net
imports System.Security.Cryptography.X509Certificates

Public Class TrustAllCertificatePolicy
    Implements System.Net.ICertificatePolicy

    Public Sub New()
        MyBase.New()

    End Sub

    Public Function CheckValidationResult(ByVal sp As ServicePoint, ByVal cert As X509Certificate, ByVal req As WebRequest, _
        ByVal problem As Integer) As Boolean Implements ICertificatePolicy.CheckValidationResult
        Return True
    End Function
End Class


